src/playground/Pages/Tabs/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Header } from "@/components/Blocks/Header"
+import { Breadcrumb } from "@/components/Navigation/Breadcrumb"
+import {
+  Tabs as TabsComponent,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from "@/components/Navigation/Tabs"
+import { User } from "@/icons"
+import React, { forwardRef } from "react"
+
+interface TabsProps {
+  tabs: TabType[]
+  title: string
+  subtitle: string
+  src: string
+  alt: string
+  defaultTab?: string
+}
+
+interface TabType {
+  name: string
+  key: string
+  content: React.ReactNode
+}
+
+export const Tabs = forwardRef<HTMLDivElement, TabsProps>(
+  ({ tabs, title, subtitle, src, alt, defaultTab = tabs[0].key }, ref) => (
+    <main ref={ref}>
+      <Breadcrumb icon={User} route={"Employees"} title={"Alba Horneros"} />
+      <Header title={title} subtitle={subtitle} src={src} alt={alt}></Header>
+      <TabsComponent defaultValue={defaultTab}>
+        <TabsList className="h-auto w-full justify-start rounded-none border-b border-l-0 border-r-0 border-t-0 border-solid border-b-muted bg-transparent px-10 py-3">
+          {tabs.map((tab: TabType) => (
+            <TabsTrigger
```

Comment:
This is really interesting. Eventually, we'll want to use a React Router `<Link>` element here, but we don't want to get coupled to it as a library. Since this uses radix behind the scenes, we can use the `asChild` prop and just pass the `<Link />` component as children and they will merge.

This is how it works: https://www.radix-ui.com/primitives/docs/utilities/slot

Reply:
What I mean is - we probably want to eventually find a way to pass down a component as a trigger?

Reply:
Agreed. I haven't experimented yet with connecting this component to Factorial, the goal here was simply creating the Layout UI, I think there's still some work left before jumping to that, but it's definitely a challenge I have in mind. Thanks for the doc.